src/Servers/Kestrel/Core/test/KestrelServerTests.cs

```diff
@@ -287,10 +293,20 @@ public void ConstructorWithNullTransportFactoryThrows()
         ILoggerFactory loggerFactory = null,
         KestrelMetrics metrics = null)
     {
+        var httpsConfigurationService = new HttpsConfigurationService();
+        if (options?.ApplicationServices is IServiceProvider serviceProvider)
+        {
+            httpsConfigurationService.Initialize(
+                serviceProvider.GetRequiredService<IHostEnvironment>(),
+                serviceProvider.GetRequiredService<ILogger<KestrelServer>>(),
+                serviceProvider.GetRequiredService<ILogger<HttpsConnectionMiddleware>>());
+        }
+	
         return new KestrelServerImpl(
             Options.Create<KestrelServerOptions>(options),
             transportFactories,
             multiplexedFactories,
+			httpsConfigurationService,
```

Comment:
```suggestion
            httpsConfigurationService,
```

Reply:
Huh, that must have been from a merge.  Thanks!